tests/phpunit/CheckUsernameTest.php

```diff
@@ -136,9 +136,9 @@ public function testDisabledChecking(): void
         $logger = new TestLogger();
         $config = $this->createConfig();
         putenv('KBC_REALUSER=' . (string) getEnv('COMMON_DB_USER'));
-        new Application($config, $logger);
+        $this->getApp($config, $logger);
 
-        Assert::assertEquals([], $logger->records);
+        Assert::assertCount(1, $logger->records);
```

Comment:
To je OK, že se teď tam je záznam a předtím ne?

Reply:
jo protože BaseComponent vyhazuje automaticky hlášku o spuštění komponenty 
https://github.com/keboola/php-component/blob/master/src/BaseComponent.php#L56

a v records jsou i debug hlášky